llvm/test/CodeGen/DirectX/legalize-i8.ll

```diff
@@ -127,10 +127,9 @@ define i32 @i8_geps_index0() {
   ; CHECK:         [[LOAD:%.*]] = load i32, ptr [[GEP]], align 4
   ; CHECK-NEXT:    ret i32 [[LOAD]]
   %1 = alloca [2 x i32], align 8
-  %2 = getelementptr inbounds nuw i8, ptr %1, i32 0
-  %3 = load i8, ptr %2
-  %4 = sext i8 %3 to i32
-  ret i32 %4
+  %2 = load i8, ptr %1
```

Comment:
Why did this test need to change? Shouldn’t we check that if a geo already exists we don’t do your pass?

Reply:
It didn't need to change, but there is already another i8 load test that has a GEP to access index 1, so I just rewrote the index 0 test to omit the GEP.

Reply:
I can restore the original test and add a completely new test instead if you prefer.